core-rust/engine-state-api-server/engine-state-api-schema.yaml

```diff
@@ -11,6 +10,12 @@ info:
     url: https://cdn.document360.io/50e78792-5410-4ac9-aa43-4612b4d33953/Images/Documentation/radix-logo-banner.png
     href: https://docs.radixdlt.com
   description: |
+    *Beta Disclaimer*
+    Please note that this API is currently in its beta phase. It is subject to breaking changes
+    without any prior notice. It should not be used for business-critical integrations before the
+    `version` indicated above becomes stable, which is expected in Q4 of 2024 - please pay attention
+    to the Radix Node's future release notes.
+    
```

Comment:
Perhaps the below? I've weakened the advisory a little. And removed "without prior notice" because that implies it's not been mentioned in the release notes.

What do you think?

```suggestion
    **This API is currently in Beta**
    This API interface may experience breaking changes as part of node releases. Such changes will be clearly mentioned in the [babylon-node release notes](https://github.com/radixdlt/babylon-node/releases).
    We advise against using this API for business-critical integrations before the `version` indicated above becomes stable, which is expected in Q4 of 2024.
    
```

Reply:
very nice, thanks 👍 

let me only fix the "API interface" (which would unroll to non-compiling "application programming interface interface")